### DISCUSSION TITLE
How and where do I edit server settings?

### DISCUSSION THREAD

**Discussion by lisboamario** [thijsvanloef/palworld-server-docker#107] [Q&A]
For example, death penalty, stamina, capture ratio, etc. I knew how to do this on the local server on my computer, but I don't know where or how to do it in a VM

**Comment by thijsvanloef** [comment]
I've just merged the ability to set server settings via Environment Variables: https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#with-environment-variables
